### ISSUE TITLE
Xvideos.com

### ISSUE THREAD

**Issue by epicsugestions** [PreMiD/PreMiD#92]
What's the name of the service?
XVideos

What should the Presence display?
Information about the current video playing, maybe who posted it too.

Why do you want support for this service?
because xvideos is the best brazilian porn website alongside ''a casa das brasilerinhas" and i will love to see the rich presence

Are you able to provide an image for the service? (512x512 minimum)
https://seeklogo.com/images/X/xvideos-logo-77E7B4F168-seeklogo.com.png

**Comment by Fruxh** [comment]
Sending this message because we are getting more and more requests.

Are you able to prove that anyone except you will use this service?

### Your service request will be added after completing at least 5-10+ :thumbsup:  votes.

``Or you donate to support this service! 
(You will be credited if you can prove that you donated.)``
<details>
<summary>Donation links</summary>
<a href="https://www.patreon.com/bePatron?u=4610890">Patreon</a><br>
<a href="https://ko-fi.com/E1E1HLLE">Ko-fi</a><br>
<a href="https://www.paypal.com/cgi-bin/webscr?cmd=_s-xclick&hosted_button_id=ZU8Q766ACS2WS&lc=US">PayPal</a><br>
BitCoin: <code>18WHUcQbThwExKhUyLGPRFUGSwFqftpsEn</code>
</details><br>
~ Fruxh

**Comment by kvq** [comment]
Yes, because I need it as well. So me and my friends can watch xvideos together

**Comment by frozaum** [comment]
> Yes, because I need it as well. So me and my friends can watch xvideos together

i need it because i want everyone to see me watching avengers infinity war in hd

**Comment by DavidAlejandroS** [comment]
**YES**
